### PR TITLE
CUMULUS-1755: Update TEA

### PR DESCRIPTION
**Pull request by markdboyd**
**Summary:** Summary of changes

Addresses [CUMULUS-1755: Update TEA to v59+](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1755)

## Changes

* Updates TEA to build 61

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Adhoc testing

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Change variable `saml_launchpad_metadata_path` to
     `saml_launchpad_metadata_url` in the `tf-modules/cumulus` Terraform module.
 
+- **CUMULUS-1755**
+  - The `thin_egress_jwt_secret_name` variable for the `tf-modules/cumulus` Terraform module is now **required**. This variable is passed on to the Thin Egress App in `tf-modules/distribution/main.tf`, which uses the keys stored in the secret to sign JWTs. See the [Thin Egress App documentation on how to create a value for this secret](https://github.com/asfadmin/thin-egress-app#setting-up-the-jwt-cookie-secrets).
```
Comment (laurenfrederick) [inline CHANGELOG.md]: @markdboyd Is this part of our deployment documentation?

Comment (markdboyd) [reply CHANGELOG.md]: No it is not, but you're right that it should probably be added

File: CHANGELOG.md [hunk 2]
```diff
@@ -25,6 +25,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
   - Change variable `saml_launchpad_metadata_path` to
```
Comment (laurenfrederick) [inline CHANGELOG.md]: This was released in 1.18.0. Maybe a bad merge? Would you mind removing this?

File: CHANGELOG.md (hunk 1 above)
Comment (markdboyd) [reply CHANGELOG.md]: 4aef75240f9ae85ef5ecaf69095f185b13aabb38

**Pull request merged**